src/Akka.Streams.Kafka/Dsl/KafkaProducer.cs

```diff
@@ -66,5 +67,11 @@ public static class KafkaProducer
                 ? flow
                 : flow.WithAttributes(ActorAttributes.CreateDispatcher(settings.DispatcherId));
         }
+
+        // TODO
+        public static FlowWithContext<IEnvelope<K, V, NotUsed>, C, DeliveryReport<K, V>, C, NotUsed> FlowWithContext<K, V, C>(ProducerSettings<K, V> settings)
```

Comment:
what do we need to do here?

Reply:
Anything from Akka.Streams itself?

Reply:
I have defined this method just to put valid reference in xml comment for consumer's static method (because it is usually used in pair with this producer method). 
So this is just something I will implement in next PRs. Did not check out if something is missing from `Akka.Streams` package, but I think all is ok here.